ezcontent.info.yml

```diff
@@ -13,6 +13,8 @@ install:
   - comment
   - config
   - contact
+  - content_lock
```

Comment:
Please remove this line as its submodule will enable parent module `content_lock`